### PR TITLE
Some new organisations mainly for France and Sweden

### PR DESCRIPTION
**Pull request by PierreMesure**
Thanks for the contribution! **Below** is a template that can make it easier when submitting your Organization:

**Your Organization**: I'm a citizen 
**GitHub Organization url**: N/A
**Country/Locality**: N/A

Inclusion requirements:
- [ X ] Referenced account is an [Organization](https://github.com/github/government.github.com#add-organization) not a User^
- [ X ] If this is a government project, your Organization's description should include a reference to your country/agency.
- [ X ] Make sure your Organization includes at least one public repository
- [ X ] Please also include a URL for your organization that links back to your organization or agency homepage.
 
^ If you want to convert your account to an org, details can be found here: https://help.github.com/articles/converting-a-user-into-an-organization/
  
 :heart: GitHub Government Contributors

### CODE REVIEW

File: _data/governments.yml [hunk 1]
```diff
@@ -198,19 +199,32 @@ Finland:
...
   - communaute-cimi
+  - culturecommunication
   - DGFiP
   - diplomatiegouvfr
   - disic
+  - entrepreneur-interet-general
   - erasme
   - etalab
+  - france-connect
+  - GouvernementFR
```
Comment (nschonni) [inline _data/governments.yml]: This is a User rather than an Org

File: _data/governments.yml [hunk 2]
```diff
@@ -198,19 +199,32 @@ Finland:
...
+  - MINAGRI-INITIAL
+  - MTES-MCT
   - nanterre
+  - nantesmetropole
   - polewebmaedi
   - ProgrammeVitam
-  - sgmap
+  - region-bretagne
   - sgmap-agd
+  - SIG-Region-Hauts-de-France
```
Comment (nschonni) [inline _data/governments.yml]: This is a User rather than an Org

File: _data/governments.yml [hunk 3]
```diff
@@ -363,18 +377,27 @@ Spain:
   - JuntaDeAndalucia
  
 Sweden:
+  - arbetsformedlingen
   - domstol
   - gbif-sweden
+  - GoteborgsStad
```
Comment (nschonni) [inline _data/governments.yml]: This is a User rather than an Org

**Review by nschonni — CHANGES_REQUESTED** [review]
Awesome, but it looks like three of these are still user accounts, that is why the build is failing

**Comment by PierreMesure** [comment]
You're right, I missed them! I removed them and will contact the one I know to see if they can change that.

**Comment by jbjonesjr** [comment]
@PierreMesure @SIG-Region-Hauts-de-France is still listed as a user, not an org.

**Comment by PierreMesure** [comment]
Done, sorry. I hope it's good now.

**Comment by nschonni** [comment]
Build failure is unrelated. I've submitted #681 to fix that

File: _data/governments.yml [hunk 4]
```diff
@@ -199,19 +200,29 @@ Finland:
...
   - erasme
   - etalab
+  - lutece-secteur-public
+  - MINAGRI-INITIAL
+  - MTES-MCT
   - nanterre
+  - nantesmetropole
   - polewebmaedi
   - ProgrammeVitam
-  - sgmap
```
Comment (nschonni) [inline _data/governments.yml:220]: This one looks like it's moved somewhere else, but the org is still there

**Review by nschonni — APPROVED** [review]

**Comment by PierreMesure** [comment]
Hey, shall we merge this branch?

**Comment by PierreMesure** [comment]
@jbjonesjr @nschonni Is there anything more to fix to merge that branch?

**Comment by jbjonesjr** [comment]
all good.

**Pull request merged**